### ISSUE TITLE
Sub-workflow manual transition do not show up in the Pipeline Dashboard

### ISSUE THREAD

**Issue by lrochette** [electriccommunity/ec-pipeline-activity#20]
I've a dashboard that shows a pipeline in progress

![activity_-_electriccommander](https://cloud.githubusercontent.com/assets/2013371/4926855/d056dd6a-6536-11e4-8074-a807f6eba919.png)

but it's not obvious this workflow is waiting for a manual transition in the sub-workflow

![workflow_details_-_pipeline_song_1_1_34-41](https://cloud.githubusercontent.com/assets/2013371/4926880/f906098e-6536-11e4-92bf-d78cfd70dcb1.png)

Would it be possible to surface that information that the workflow is waiting for something?

**Comment by tanaynagjee** [comment]
Yes, it's a good use case.  The current design has not taken subworkflows into consideration at all.
